src/tests/test_bookmark_endpoints.py

```diff
@@ -0,0 +1,148 @@
+from http import HTTPStatus
+
+from starlette.testclient import TestClient
+from database.session import DbSession
+
+from tests.testutils.users import logged_in_user, ALICE
+from database.model.bookmark.bookmark import Bookmark
+from database.session import DbSession
+from tests.testutils.users import register_asset
+from datetime import datetime
+from database.model.agent.person import Person
+from database.model.agent.contact import Contact
+from database.authorization import User
+
+def test_create_bookmark(
+    client: TestClient,
+    person: Person) -> None:
+
+    register_asset(person)
+    response = client.get("/persons")
+    identifier = response.json()[0]["identifier"]
+
+    with logged_in_user():
+        response = client.post(
+            "/bookmarks",
+            json={"resource_identifier": identifier},
+            headers={"Authorization": "fake token"},
+        )
+    assert response.status_code == HTTPStatus.OK
+    bookmark = response.json()
+    assert bookmark["resource_identifier"] == identifier
+    assert bookmark["created_at"] is not None
```

Comment:
```suggestion
    assert bookmark["created_at"] is not None
```
We know the bookmark was created just now, we probably want to check this reflects that. E.g., within the last 2 seconds should be a very safe check.